nixos/modules/services/wayland/cage.nix

```
@@ -36,8 +36,18 @@ in {
     type = types.path;
     default = "${pkgs.xterm}/bin/xterm";
     defaultText = literalExpression ''"''${pkgs.xterm}/bin/xterm"'';
+    example = literalExpression ''
+      pkgs.writeShellScript "cage-program" \'\'
+        cd /home/${user}
```

error [log](https://github.com/NixOS/nixpkgs/actions/runs/7565478740/job/20601302123?pr=281681):

> error: undefined variable 'user'

this needs to be escaped

```suggestion
        cd /home/''${user}
```